docs/service-catalog/06-01-service-binding-usage.md

```diff
@@ -83,5 +113,5 @@ These components use this CR:
 
 | Component   |   Description |
 |----------|------|
-| Service Binding Usage Controller |  Reacts to every action of creating, updating, or deleting ServiceBindingUsages in all Namespaces, and uses ServiceBindingUsage data to inject binding. |
-| Console Backend Service |  Exposes the given CR to the Console UI. It also allows you to create and delete a ServiceBindingUsage. |
+| [ServiceBindingUsage Controller](https://github.com/kyma-project/kyma/tree/master/components/service-binding-usage-controller) |  Reacts to every action of creating, updating, or deleting ServiceBindingUsage resources in all Namespaces, and uses ServiceBindingUsage data to inject binding. |
```

Comment:
```suggestion
| [ServiceBindingUsage Controller](https://github.com/kyma-project/kyma/tree/master/components/service-binding-usage-controller) |  Reacts to every action of creating, updating, or deleting ServiceBindingUsage resources in all Namespaces and uses ServiceBindingUsage data to inject binding. |
```

Oxford comma is only used when there are more than two elements in a list. See the [Oxford comma definition](https://www.lexico.com/definition/oxford_comma).